PublicMapping/districtbuilder
# Style home screen / map list

## Overview
This PR styles the map list and empty state on the Home screen. 

### Demo
<img width="1636" alt="Screen Shot 2020-08-16 at 12 04 04 PM" src="https://user-images.githubusercontent.com/5672295/90333434-ac416000-dfbd-11ea-8437-ed987f6d832f.png">
<img width="1636" alt="Screen Shot 2020-08-16 at 12 04 26 PM" src="https://user-images.githubusercontent.com/5672295/90333439-b1061400-dfbd-11ea-9921-f8b4da077b98.png">
<img width="1636" alt="Screen Shot 2020-08-16 at 12 04 30 PM" src="https://user-images.githubusercontent.com/5672295/90333440-b19eaa80-dfbd-11ea-87b0-661557fe5e9d.png">
<img width="1636" alt="Screen Shot 2020-08-16 at 12 04 47 PM" src="https://user-images.githubusercontent.com/5672295/90333441-b2374100-dfbd-11ea-9b87-377f18346dee.png">

### Notes
- I was having trouble figuring out how to make `rc-menu` keyboard accessible, and came across `react-aria-menubutton` [(link)](https://github.com/davidtheclark/react-aria-menubutton), which gives us a seemingly fully accessible menu for free - sans extra class names and styles. 
- I did up this little illustration for the empty state, and wrote up some introductory text that I'm not 100% confident about. Open to any and all feedback on either.
- Since there's a lot less info here than on the wireframes, I did not make this look like a table. As with the above, open to any feedback.

## Testing Instructions
- `git pull`
- Run yarn install.
- Log in.
- You should be on the map view page.
- If you have maps, hover over them and click to see how that works.
- In `HomeScreen.tsx`, comment out lines `250` through `283` and line `313`.
- You should now see the empty state.
- Now, test out the profile menu. You should be able to click on Logout to log out, OR use your keyboard for the same result.

Closes #274
